isthmus/build.gradle.kts

```diff
@@ -29,6 +29,7 @@ dependencies {
   implementation("com.google.code.findbugs:jsr305:3.0.2")
   implementation("com.github.ben-manes.caffeine:caffeine:3.0.4")
   implementation("org.immutables:value-annotations:2.8.8")
+  testImplementation("org.apache.calcite:calcite-plus:1.28.0")
```

Comment:
Do we need this new dependency?

Reply:
Yes I agree about your point on dependencies.
But
- this is only a test dependency.
- It has scale factor Stats.
- Down the line if we want to develop the Query Generator the SF and stats maybe useful...

But
- Above are not very compelling, I know.
- Immediate hurdle is I have to figure out how to export the DDL script from the Tables provided by TpcdsSchema.
   Let me know if you have any pointers on this.

@guykhazma you ok with removing TpcdsSchema. You were suggesting going the opposite way of doing the same for Tpch.

Reply:
If the class is used by any code, it will be included in the Graal image, whether it is referred to from Isthmus or just from code we reference (e.g. internal Calcite classes). I'd guess that Schema falls into that category.

Reply:
Based on this have left the test dependency in the updated patch.